e2e/config/vhd.go

```diff
@@ -237,15 +237,15 @@ func replicateToCurrentRegion(ctx context.Context, t *testing.T, definition sigI
 	if err != nil {
 		return fmt.Errorf("begin updating image version target regions: %w", err)
 	}
-	if _, err := resp.PollUntilDone(ctx, nil); err != nil {
+	if _, err := resp.PollUntilDone(ctx, DefaultPollUntilDoneOptions); err != nil {
 		return fmt.Errorf("updating image version target regions: %w", err)
 	}
 
 	return nil
 }
 
 func ensureProvisioningState(version *armcompute.GalleryImageVersion) error {
-	if *version.Properties.ProvisioningState != armcompute.GalleryImageVersionPropertiesProvisioningStateSucceeded {
+	if *version.Properties.ProvisioningState != armcompute.GalleryProvisioningStateSucceeded {
```

Comment:
are these actually the same? I personally don't see the point in changing this since `version` is a gallery image version rather than a gallery

Reply:
I upgrade armcompute from v1 to v6. I fixed compilation errors here, original constants are gone